extensions/2.0/Khronos/KHR_interactivity/Specification.adoc

```diff
@@ -0,0 +1,2914 @@
+.. Remove this activation index value from the node's and the graph's arrays of activation indices.
+.. Cancel the corresponding scheduled activation.
+
+===== Cancel Delay
+
+[cols="1h,1,2"]
+|===
+| Type | `flow/cancelDelay` | Cancel a previously scheduled output flow activation
+| Input flow sockets
+| `in` | The entry flow into this node
+| Input value sockets
+| `int delayIndex` | The index value of the scheduled activation to be cancelled
+| Output flow sockets
+| `out` | The flow to be activated after executing this node
+|===
+
+This node has no internal state but its execution **MAY** affect internal states of other nodes and the graph.
+
+When the `in` input flow is activated:
+
+1. Evaluate the `delayIndex` input value.
+2. Remove this activation index value from all arrays of activation indices if it exists.
+3. Cancel the corresponding scheduled activation if it exists.
+4. Activate the `out` output flow.
+
+Non-positive or not existing delay index values **MUST NOT** cause any runtime errors.
+
+=== State Manipulation Nodes
+
+==== Custom Variable Access
+
+===== Variable Get
+
+[cols="1h,1,2"]
+|===
+| Type | `variable/get` | Get a custom variable value
+| Configuration
+| `int variable` | The custom variable index
+| Output value sockets
+| `T value`  | The custom variable value
+|===
+
+This node gets a custom variable value using the variable index provided by the `variable` configuration value. The type `T` is determined by the referenced variable. The variable index **MUST** be non-negative and less than the total number of custom variables, otherwise the node is invalid.
+
+This node has no internal state.
+
+===== Variable Set
+
+[cols="1h,1,2"]
+|===
+| Type | `variable/set` | Set a custom variable value
+| Configuration
+| `int variable` | The custom variable index
+| Input flow sockets
+| `in` | The entry flow into this node
+| Input value sockets
+| `T value` | The new variable value
+| Output flow sockets
+| `out` | The flow to be activated after the value is set
+|===
+
+This node sets a custom variable value using the variable index provided by the `variable` configuration value and the `value` input value. The type `T` is determined by the referenced variable. The variable index **MUST** be non-negative and less than the total number of custom variables, otherwise the node is invalid.
+
+This node has no internal state.
+
+When the `in` input flow is activated:
+
+1. Evaluate the `value` input value.
+2. Set the custom variable with the `variable` configuration value index to the `value` input value.
+3. Activate the `out` output flow.
+
+==== Object Model Access
+
+===== Pointer Get
+
+[cols="1h,1,2"]
+|===
+| Type | `pointer/get` | Get an object model property value
+| Configuration
+| `string pointer` | The JSON pointer or JSON pointer template
+| Input value sockets
+| `int <segment>` | The JSON pointer template path segment to be substituted at runtime
```

Comment:
Since input value sockets MUST be connected, is `segment` optional socket(s)? or its omnipresent ? What if pointer is not templated, what shall `socket` be connected to, random const value (like 0) ?

Reply:
I think I replied to myself below, so #solved